src/components/ScholarshipForm.jsx

```diff
@@ -23,9 +24,8 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function ScholarshipForm() {
+function ScholarshipForm({ setSubmissionAlert }) {
   const classes = useStyles();
-  const [submissionAlert, setSubmissionAlert] = useState(null);
   const user = firebase.auth().currentUser;
 
   const formik = useFormik({
```

Comment:
(For future reference) I'm realizing that the `formik` value maybe doesn't need to be passed in as a property. We could just pass in a **ScholarshipModel** that comes from either **Scholarships.id()** or **Scholarships.new()** and get the default/initial form values from there.

Reply:
interesting!